ajax/libs/antiscroll/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "antiscroll",
+  "filename": "antiscroll.min.js",
+  "version": "0.9",
+  "description": "Antiscroll addresses this issue by providing a cross-browser implementation of the scrollbars popularized by OS X Lion that retains native properties.",
+  "author": {
+    "name": "James Nylen",
+    "url": "https://github.com/nylen"
+  },
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "http://github.com/nylen/antiscroll"
```

Comment:
@pvnr0082t, sorry for making this mistake, I mistook the `url` in `repository` field, and I modified `url`, thanks for your help!